darwin/exporter/formats/nifti.py

```diff
@@ -137,50 +137,28 @@ def check_for_error_and_return_imageid(
     image_id : str
 
     """
-    # check if all item slots have the correct file-extension
+    # Check if all item slots have the correct file-extension
     for slot in video_annotation.slots:
         for source_file in slot.source_files:
             filename = Path(source_file["file_name"])
-
-            try:
-                suffixes = filename.suffixes[-2:]
-            except IndexError:
-                suffixes = filename.suffixes
-            if len(suffixes) == 2:
-                if suffixes[0] == ".nii" and suffixes[1] == ".gz":
-                    image_id = str(filename).rstrip("".join(suffixes))
-                else:
-                    return create_error_message_json(
-                        "Two suffixes found but not ending in .nii.gz",
-                        output_dir,
-                        str(filename),
-                    )
-            elif len(suffixes) == 1:
-                if suffixes[0] == ".nii" or suffixes[0] == ".dcm":
-                    image_id = filename.stem
-                else:
-                    return create_error_message_json(
-                        "Misconfigured filename, not ending in .nii or .dcm. Are you sure this is medical data?",
-                        output_dir,
-                        str(filename),
-                    )
-            else:
+            if not (
+                filename.name.endswith(".nii.gz")
```

Comment:
Can we also make sure we check that case-insensitive? That's gonna be next reported bug.

Reply:
Updated, all 6 checks are now case-insensitive